pkg/rendering/helm.go

```diff
@@ -0,0 +1,61 @@
+package rendering
+
+import (
+	"fmt"
+
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chartutil"
+	"helm.sh/helm/v3/pkg/engine"
+)
+
+// HelmEngine is the helm engine constant to identify the engine type
+const HelmEngine = "helm"
+
+// HelmRenderingEngine is a rendering engine based on helm which
+// satisfies the RenderingEngine interface
+type HelmRenderingEngine struct{}
+
+// NewHelmRenderingEngine returns a new Helm engine
+func NewHelmRenderingEngine() Engine {
+	return HelmRenderingEngine{}
+}
+
+// Render renders the template interpolating the supplied values
+func (h HelmRenderingEngine) Render(content string, values map[string]interface{}) string {
+	out := h.RenderFiles([]*chart.File{
+		{Name: "templates", Data: []byte(content)},
+	}, map[string]interface{}{}, values)
+
+	return out["templates"]
+}
+
+// RenderMap renders the values passed from a map[string]string
+func (h HelmRenderingEngine) RenderMap(contentMap map[string]string, values map[string]interface{}) map[string]string {
+	for k, v := range contentMap {
+		contentMap[k] = h.Render(v, values)
+	}
+	return contentMap
+}
+
+// RenderFiles render files as they where charts with helm
```

Comment:
Maybe move doc to interface, this renders each entry in template with the passed in values.